OMCompiler/Compiler/FrontEnd/NFSCodeCheck.mo

```diff
@@ -117,14 +117,14 @@ algorithm
         replaceablePrefix = SCode.REPLACEABLE()))), _)
       then ();
 
-    //case (NFSCodeEnv.CLASS(cls = SCode.CLASS(name = name, prefixes = SCode.PREFIXES(
-    //    replaceablePrefix = SCode.NOT_REPLACEABLE()), info = info)), _)
-    //  equation
-    //    Error.addSourceMessage(Error.ERROR_FROM_HERE, {}, inOriginInfo);
-    //    Error.addSourceMessage(Error.NON_REPLACEABLE_CLASS_EXTENDS,
-    //      {name}, info);
-    //  then
-    //    fail();
+    case (NFSCodeEnv.CLASS(cls = SCode.CLASS(name = name, prefixes = SCode.PREFIXES(
```

Comment:
This was disabled for a reason, see 4a763fb2424f37a22bcaa3f1a210812b3c1d1c54. The whole case should've been removed rather than just commented out though.

Reply:
Adjusted. Putting it as resolved:)